specification/window.browser.bs

```diff
@@ -52,6 +52,39 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     it MUST be used only for WebExtension related functionality.
   </section>
 
+  <section>
+    For backwards compatibility with older versions of web extensions, a UA MAY
+    alias individual properties of {{browser}} to <code>chrome</code>. {{browser}}
```

Comment:
Can we say something like "to an exposed global with the identifier chrome"? Since we haven't defined chrome here and it is unclear what it is.

Reply:
It is mentioned in the non normative portion at the beginning. But happy to add that